### PR TITLE
Issue-442: Add ms_machine_id and s1_agent_id observable types

### PR DESCRIPTION
**Pull request by halcyon**
Necessary to add support for third party host ID observables - https://github.com/threatgrid/hydrant/issues/442

This `adds ms_machine_id` and `s1_agent_id` observable types.

### CODE REVIEW

File: src/ctim/schemas/vocabularies.cljc [hunk 1]
```diff
@@ -255,6 +255,8 @@
     "imei"
     "imsi"
     "amp_computer_guid"
+    "ms_machine_id"
+    "s1_agent_id"
```
Comment (msprunck) [inline src/ctim/schemas/vocabularies.cljc:259]: Could you add a comment with a description or a reference link? What is a `ms_machine_id`? `s1_agent_id`?

**Review by msprunck — APPROVED** [review]
LGTM!

File: src/ctim/schemas/vocabularies.cljc (hunk 1 above)
Comment (halcyon) [reply src/ctim/schemas/vocabularies.cljc:259]: Thank you - I don't know much about these beyond what I've managed to determine by examining the examples in the issue. 

Here are my notes:
> Microsoft Windows Defender machine IDs appear to be 40 character hexadecimal strings
> Sentinel One Agent IDs appear to be 18 character decimal strings

As I'm far from an authoritative source, would it make sense to add these as a comment to `vocabularies.cljc`?

Comment (halcyon) [reply src/ctim/schemas/vocabularies.cljc]: Will do!

**Pull request merged**